src/css/footer.css

```diff
@@ -0,0 +1,207 @@
+.footer {
+  padding-top: 40px;
+  padding-bottom: 40px;
+  background-color: var(--color-dark-green);
+}
+

```

Comment:
@media screen and (min-width: 768px) {
  .footer {
    padding-top: 80px;
  }
}

@media screen and (min-width: 1280px) {
  .footer {
    padding-top: 104px;
    padding-bottom: 32px;
  }
}